build-logic/packaging/build.gradle.kts

```diff
@@ -16,4 +16,6 @@ dependencies {
     implementation("com.google.code.gson:gson")
 
     testImplementation("org.junit.jupiter:junit-jupiter")
+
+    testImplementation(gradleTestKit())
```

Comment:
Isn't this there automatically since we use the `java-gradle-plugin`?

Reply:
Missed that, thanks!